src/components/ImageWithSizeCalculation.js

```diff
@@ -39,16 +48,18 @@ const defaultProps = {
  *
  */
 function ImageWithSizeCalculation(props) {
-    const isLoadedRef = useRef(null);
+    const [loadState, setLoadState] = useState(LoadState.INITIAL);
     const [isImageCached, setIsImageCached] = useState(true);
-    const [isLoading, setIsLoading] = useState(false);
-
+    const source = useMemo(() => ({ uri: props.url }), [props.url]);
+    const {isOffline} = useNetwork();
```

Comment:
Please explain which we need to change because if we change only,
`const {isOffline} = useNetwork();`

this line then it is okay but I don't understand why we need to replace this line,

`const source = useMemo(() => ({ uri: props.url }), [props.url]);`

it is being used for displaying image